master/internal/db/postgres_experiments.go

```diff
@@ -884,9 +884,6 @@ func (db *PgDB) DeleteExperiments(ctx context.Context, ids []int) error {
 	if err != nil {
 		return errors.Wrapf(err, "error deleting experiments %v", ids)
 	}
-	if len(deletedIDs) != len(ids) {
```

Comment:
weirdly enough, this even sort of caused a bug. when we tried to delete a single experiment and it was missing, we then tried to move it to delete failed because of this error, and that printed an even more confusing error